### PR TITLE
Range slider for gas price

### PR DESCRIPTION
**Pull request by c-johnson**
Left it with the default styling for now.

### CODE REVIEW

**Comment by Fang-** [comment]
>gas price in eth, not gwei

Ehh, it's pretty hard to count the eight zeroes in the `eth` notation for `1 gwei`. Because these are such small units, gwei is super common. I think we should stick with it.

A solution that would still inform the user of actual, tangible value, would be to display the `gas price * gas limit` to them as "max tx fee in eth". This usually ends up being something with like two or three zeroes? Much more readable.

File: src/components/StatelessTransaction.js [hunk 1]
```diff
@@ -59,6 +63,31 @@ const StatelessTransaction = props => {
...
   })
 
+  const gasPriceRangeDialogue = (
+    <React.Fragment>
+      <div className="mt-8">Gas Price: <span className="ml-4 text-700 text-sm">{(gasPrice / 1000000000).toFixed('9')} eth</span></div>
+
+      <input
+        className="mt-4"
+        type="range"
+        min="4"
```
Comment (Fang-) [inline src/components/StatelessTransaction.js]: imo this should be able to go as low as 2 gwei.

Comment (c-johnson) [reply src/components/StatelessTransaction.js]: Roger.

**Comment by c-johnson** [comment]
The task recommended to change it to eth, but I agree. Added your revisions.

File: src/components/StatelessTransaction.js [hunk 2]
```diff
@@ -84,6 +113,12 @@ const StatelessTransaction = props => {
       </InnerLabel>
     </Input>
 
+  const maxTxnPriceInEth = (
+    <div className="mt-8">
+      Max transaction price: <span className="text-700">{(gasPrice * gasLimit) / 1000000000} eth</span>
```
Comment (Fang-) [inline src/components/StatelessTransaction.js]: "Transaction *fee*" is canonical terminology here. I recommend changing that in the variable name also, just to stay consistent.

Comment (Fang-) [reply src/components/StatelessTransaction.js]: I'd also give this a slightly smaller top margin, `8` is a bit much. `2` might look best to keep it nice and tied to the slider.

Comment (c-johnson) [reply src/components/StatelessTransaction.js]: Moved max transaction fee to the top right of the slider; what do you think?

**Pull request merged**